specification/terms.md

```diff
@@ -106,14 +109,51 @@ fn main() {
 }
 ```
 
-### §2.4 Note on the terms lvalue and rvalue
+### §2.4 Use vexpr
+
+```Rust
+    VExpr::Use(LExpr)
+```
+
+An implicit vexpr that wraps a lexpr when a vexpr is expected.
```

Comment:
But my mental model is an ADT model and I want to keep it. For prior art, the C standard *also* has lvalue-to-rvalue conversions.